assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js

```diff
@@ -17,17 +17,6 @@ export class KeycloakLoader {
         const msg = "Cannot load keycloak settings. This is normal for single-user mode.";
 
         return new Promise((resolve, reject) => {
-            try {
-                if (window.parent && window.parent['_keycloak']) {
-                    window['_keycloak'] = window.parent['_keycloak'];
-                    resolve(window['_keycloak']);
-                    return;
-                }
```

Comment:
are we excluding this part?

Reply:
As I understand this was implemented for old UD which set that object
https://github.com/eclipse-che/che-dashboard/blob/7.11.0/src/app/index.module.ts#L205. 

The ability to get global data from the parent frame was closed for security reasons. That is why the current UD doesn't provide this object 'window['_keycloak']' at all.